editor/animation_editor.cpp

```diff
@@ -1092,12 +1092,16 @@ void AnimationKeyEditor::_track_editor_draw() {
 	int sep = get_constant("vseparation", "Tree");
 	int hsep = get_constant("hseparation", "Tree");
 	Color color = get_color("font_color", "Tree");
-	Color sepcolor = Color(1, 1, 1, 0.2);
-	Color timecolor = Color(1, 1, 1, 0.2);
-	Color hover_color = Color(1, 1, 1, 0.05);
-	Color select_color = Color(1, 1, 1, 0.1);
+	Color sepcolor = color;
+	sepcolor.a = 0.2;
+	Color timecolor = color;
+	timecolor.a = 0.2;
+	Color hover_color = color;
+	hover_color.a = 0.05;
+	Color select_color = color;
+	select_color.a = 0.1;
 	Color invalid_path_color = get_color("error_color", "Editor");
-	Color track_select_color = get_color("highlight_color", "Editor");
```

Comment:
i dont think this should be changed

Reply:
you mean we should continue using the highlight color?
if so we really need to change the highlight color used in the grey theme!
But you could be right... it makes sense since those tracks than are selected... so they should be highlighted or accented ;)